master/reference/advanced/etcd-rbac/kubernetes.md

```diff
@@ -25,19 +25,14 @@ data.
 The following components need certificates with a Common Name that matches an
 etcd user that has been given appropriate roles allowing access to the key
 prefixes or paths listed below.
+
 - kube-apiserver
-  - Read and write access to `/registry`.
-    - Kubernetes v1.6 uses the etcd v3 API; this can be switched
-	  to etcd v2 by passing a flag to kube-apiserver.
-	- This key prefix can be changed by passing the proper option to the
-	  kube-apiserver.
+  - Read and write access to `/registry/`.
   - The etcd user needs to be given the root role to perform compaction when
     using the etcd v3 API (this also means that Kubernetes will have
     full read and write access to v3 data).
 - {{site.prodname}}
-  - Read and write access to `/calico`.
-  - {{site.prodname}} uses the etcd v2 API so the user and roles must be created there
-    too.
+  - Read and write access to `/calico/`.
```

Comment:
Are you sure the trailing slash won't cause any problems? It has been a while since I worked with etcd paths but I have a vague memory of some behavior differences with the trailing slash and without, though that could have been a etcd v2 specific thing.

Reply:
I believe for etcdv3 this is fine